src/rt/util/hash.d

```diff
@@ -95,15 +90,18 @@ size_t hashOf( const(void)* buf, size_t len, size_t seed )
     return hash;
 }
 
-// Check that hashOf works with CTFE
-@nogc nothrow pure unittest
```

Comment:
Having attributed unittests still has value (even when they don't verify attribute inference), because it makes it easier to completely replace one implementation with another and ensure that interface and behavior is unchanged, i.e. prevents breaking changes.
`debug` would help with `pure` and `printf` is already `nothrow` and `@nogc`, so only `@safe is problematic`, but could be easily worked around with a trusted lambda. But TBH I don't think `printf` adds any value (even for future maintenance) because its trivial.